docs/ref/validators.txt

```diff
@@ -235,32 +235,60 @@ to, or in lieu of custom ``field.clean()`` methods.
 
 .. class:: MaxValueValidator(limit_value, message=None)
 
+    :param limit_value: Value or callable.
+    :param message: If not ``None``, overrides message.
+
     Raises a :exc:`~django.core.exceptions.ValidationError` with a code of
     ``'max_value'`` if ``value`` is greater than ``limit_value``.
+    
+    .. versionchanged:: 2.2
+    
+        limit_value could be a callable.
```

Comment:
Double backtip `limit_value` and I think that _Added support for callable `limit_value` would be more appropriate? I'm not a native speaker myself so I'll let one chime in.